src/_data/starters/roxo-eleventy.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/StaticMania/roxo-eleventy",
+	"name": "roxo-eleventy",
+	"description": "Roxo is an 11ty starter project for creative agencies, freelancers, graphic designers, photographers",
+	"author": "staticmania",
```

Comment:
I believe this should be a Twitter username. Is this OK, or do you want to change it to "static_mania"?

Reply:
Yes, I want to change it to "static_mania".